src/modules/showScore.js

```diff
@@ -0,0 +1,21 @@
+const showScore = () => {
+  const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/wUcdYlqfnVytiFkqhyG8/scores/';
+
+  fetch(url, {
+    method: 'GET',
+  })
+    .then((data) => data.json())
+    .then((response) => {
+      const scoreList = document.getElementById('scoreList');
+      scoreList.innerHTML = '';
+      response.result.forEach((score) => {
+        const li = document.createElement('li');
+        li.textContent = `User: ${score.user}, Score: ${score.score}`;
+        scoreList.appendChild(li);
+      });
+    })
+    .catch((error) => {
+      throw error;
+    });
+};
```

Comment:
- [ ] Apparently, your GET request is unable to retrieve data from the API endpoint at the moment. Kindly check it and see what could be wrong.
See the [video here](https://watch.screencastify.com/v/7TY7b9kkd585gKGrxLbA)